VTCareService/src/main/java/edu/vt/cs/vtcare/vtcareservice/api/VTCareResource.java

```diff
@@ -0,0 +1,23 @@
+package edu.vt.cs.vtcare.vtcareservice.api;
+
+import edu.vt.cs.vtcare.vtcareservice.models.Provider;
+import edu.vt.cs.vtcare.vtcareservice.services.ProviderService;
+
+import javax.ws.rs.*;
+import javax.ws.rs.core.MediaType;
+
+@Path("/")
+public class VTCareResource {
+    @POST
+    @Produces(MediaType.APPLICATION_JSON)
+    @Consumes(MediaType.APPLICATION_JSON)
+    @Path("providers")
+    public Provider createProvider(Provider provider) throws Exception {
+        try {
+            ProviderService providerService = new ProviderService();
```

Comment:
Instead of creating instances manually, we could use DI and inject it directly into the controller.

Reply:
Use case is not justified for using di. Please refer literature when it is used. Technologies are used to solve a particular use case. if they are used out of place they serve to make your tasks worse. 
Also, we didn't decide to use di in the project I am not sure what do you mean by out of context comments like that.